src/create/Form.js

```diff
@@ -61,28 +62,35 @@ export default function Form({ data, onSubmit, onInputChange }) {
       </StyledInputArea>
       <StyledInputAreaDates>
         <StyledSectionFrom>
-          <StyledLabel for="schedule__input__from">Zeitraum (von)</StyledLabel>
+          <StyledLabel htmlFor="schedule__input__from">
```

Comment:
wofuer steht htmlFor? Vielleicht anderen Namen waehlen

Reply:
react variante für "for" welches die beziehung zwischen label und input über for und id referenzierung regelt